src/widgetresize/resizerstate.js

```diff
@@ -146,6 +141,23 @@ export default class ResizeState {
 		this.proposedHandleHostWidth = newSize.handleHostWidth;
 		this.proposedHandleHostHeight = newSize.handleHostHeight;
 	}
+
+	/**
+	 * Calculates a relative width of a `domResizeHost` compared to it's parent in percents.
+	 *
+	 * @protected
+	 * @param {HTMLElement} domResizeHost
+	 * @returns {Number}
+	 */
+	_getParentPercentage( domResizeHost ) {
+		const rect = new Rect( domResizeHost );
+		const domResizeHostParent = domResizeHost.parentElement;
+		// Need to use computed style as it properly excludes parent's paddings from the returned value.
+		const parentWidth = parseFloat( domResizeHostParent.ownerDocument.defaultView.getComputedStyle( domResizeHostParent ).width );
+
+		// Round to two digits in fraction.
+		return Math.round( rect.width / parentWidth * 10000 ) / 100;
```

Comment:
Rounding here is unnecessary. We only need to round saved and displayed values. The internal state can use full resolution.